platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/addressbook/RosterTestUtils.java

```diff
@@ -68,4 +70,50 @@ public static Roster addRandomRosterEntryToRoster(
         builder.rosterEntries(rosterEntries);
         return builder.build();
     }
+
+    /**
+     * Removes the node associated with the specified node ID from the roster. This does NOT create a new roster, but
+     * modifies the specified one.
+     *
+     * @param roster the roster to modify
+     * @param nodeId the ID of the node to remove
+     * @return true if the node was found and removed, else false
+     */
+    public static boolean removeNode(@NonNull final Roster roster, final long nodeId) {
+        Objects.requireNonNull(roster, "roster");
+
+        final Iterator<RosterEntry> it = roster.rosterEntries().iterator();
+        while (it.hasNext()) {
+            final RosterEntry node = it.next();
+            if (node.nodeId() == nodeId) {
+                it.remove();
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * Replace a node in the specified roster with the provided entry.
+     *
+     * @param roster the roster to modify
+     * @param node the replacement node
+     * @return true if the roster contained a node with the same node ID as the specified replacement node and it was
+     * replaced, else false
+     */
+    public static boolean replaceNode(@NonNull final Roster roster, @NonNull final RosterEntry node) {
+        Objects.requireNonNull(roster, "roster");
+        Objects.requireNonNull(node, "node");
```

Comment:
Please find a way to re-implement the tests w/o using this functionality. Or if that's totally impossible, then we could discuss this somewhere (the platform slack channel perhaps?) and maybe nuke the tests that are no longer relevant/supported.